allauth/socialaccount/migrations/0001_initial.py

```diff
@@ -12,6 +12,7 @@ class Migration(migrations.Migration):
     dependencies = (
         [
             ("sites", "0001_initial"),
+            migrations.swappable_dependency(settings.AUTH_USER_MODEL),
         ]
         if app_settings.SITES_ENABLED
         else []
```

Comment:
I don't think this is correct. The user model is always a dependency, and the site model conditionally is. So, the original code is almost correct except for missing some brackets:

```
    dependencies = (
        [
            ("sites", "0001_initial"),
        ]
        if app_settings.SITES_ENABLED
        else []
    ) + [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]
```